pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1420,7 +1448,8 @@ def _update_container_info(self, use_dev):
             self._device_container_size = SIZE_POLICY_AUTO
 
         self._device_container_raid_level = self._device_container_raid_level \
-           or defaultContainerRaidLevel(devicefactory.get_device_type(use_dev))
+                                            or defaultContainerRaidLevel(
+            devicefactory.get_device_type(use_dev))
```

Comment:
Could you please reformat this piece. Maybe to use temp variable? It's really not much readable this way.

Reply:
Fixed.